SignalMessaging/ViewControllers/ApprovalFooterView.swift

```diff
@@ -127,19 +127,36 @@ public class ApprovalFooterView: UIView {
     }()
 
     lazy var proceedButton: OWSButton = {
-        let button = OWSButton.sendButton(imageName: proceedImageName) { [weak self] in
+		let button = OWSButton.sendButton(
+			imageName: self.approvalMode.proceedButtonImageName
+		) { [weak self] in
             guard let self = self else { return }
             self.delegate?.approvalFooterDelegateDidRequestProceed(self)
         }
 
         return button
     }()
 
-    private var proceedImageName: String {
-        return approvalMode == .send ? "send-solid-24" : "arrow-right-24"
-    }
-
     private func updateContents() {
-        proceedButton.setImage(imageName: proceedImageName)
+		proceedButton.setImage(imageName: approvalMode.proceedButtonImageName)
+		proceedButton.accessibilityLabel = approvalMode.proceedButtonAccessibilityLabel
     }
 }
+
+fileprivate extension ApprovalMode {
+	var proceedButtonAccessibilityLabel: String {
+		switch self {
+		case .next: return CommonStrings.nextButton
+		case .send: return MessageStrings.sendButton
+		}
+	}
+
+	var proceedButtonImageName: String {
+		switch self {
+		case .next:
+			return "arrow-right-24"
+		case .send:
+			return "send-solid-24"
+		}
+	}
+}
```

Comment:
This is an optional tidying bit, but: since these strings are only based on the enum-case, and not the larger state of the `ApprovalFooterView`, it makes sense to have them live as vars on the enum itself.

This is totally an optional piece of the diff!